src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
any reason why you're removing a testing module in favour of the real thing? won't this make api calls during tests?

Reply:
@paddybasi - good spot! I'll swap for `HttpClientTestingModule` instead.